src/test/java/org/example/testflows/OrangeHRMTestFlow.java

```diff
@@ -211,4 +211,12 @@ public void searchLeaveListTest()
         LeavePage leavePage=new LeavePage(driver);
         leavePage.searchLeaveList(driver);
     }
-}
+
+    @Test(enabled = true,priority = 29)
+    public void validateTimePageTest() {
+        TimePage timePage=new TimePage(driver);
+        timePage.validateTimePage(driver);
+
```

Comment:
How are you validating all these things without logging into the application using :

 LoginPage loginPage=new LoginPage(driver);
        loginPage.successfulLogin(driver);

I thought you were logging into the application in each test method and perform other actions. Did you add login function in TimePage class?